components/org.wso2.carbon.identity.api.server.configs/org.wso2.carbon.identity.api.server.configs.v1/src/main/java/org/wso2/carbon/identity/api/server/configs/v1/core/ServerConfigManagementService.java

```diff
@@ -902,4 +908,121 @@ private static String includeData(Constants.ErrorMessage error, String data) {
         }
         return message;
     }
+
+    public JWTValidatorConfig getPrivateKeyJWTValidatorConfiguration() {
+
+        String tenantDomain = ContextLoader.getTenantDomainFromContext();
+        try {
+            if (JWTAuthenticationMgtOGSiServiceFactory.getInstance() != null) {
+
+                return new JWTValidatorConfig()
+                        .enableTokenReuse(JWTAuthenticationMgtOGSiServiceFactory.getInstance()
+                                .getPrivateKeyJWTClientAuthenticatorConfiguration(tenantDomain).isEnableTokenReuse());
+            } else {
+                throw new JWTClientAuthenticatorException(ERROR_JWT_AUTHENTICATOR_SERVICE_NOT_FOUND.message(),
+                        ERROR_JWT_AUTHENTICATOR_SERVICE_NOT_FOUND.code());
+            }
+        } catch (Exception e) {
```

Comment:
Can we catch specific exceptions rather catching the generic Exception

Reply:
Specific exceptions contain some exceptions from the connecter class.
If we want to catch them we need to import them.
This may point to class not found error if the connector not exists.